cloud: TiDB Serverless supports PITR (#13976)
pingcap/docs: tidb-cloud/backup-and-restore-serverless.md

1. Navigate to the **Backup** page of a cluster.

2. Click **Restore**. The setting window displays.

3. In **Restore Mode**, you can choose to restore from a specific backup or any point in time.

    <SimpleTab>
    <div label="Basic Snapshot Restore">

    To restore from a selected backup snapshot, take the following steps:

    1. Click **Basic Snapshot Restore**.
    2. Select the backup snapshot you want to restore to.

    </div>

>     <div label="Point-in-Time Restore">

Author: @pingyu PTAL

    > **Note:**
    >
    > The **Point-in-Time Restore** feature is currently in beta.

>     To restore from any point in time within the backup retention, take the following steps:

Reviewer: Within 90 days, and not before the cluster creation time.

1. Navigate to the **Backup** page of a cluster.

2. Click **Restore**. The setting window displays.

3. In **Restore Mode**, you can choose to restore from a specific backup or any point in time.

    <SimpleTab>
    <div label="Basic Snapshot Restore">

    To restore from a selected backup snapshot, take the following steps:

    1. Click **Basic Snapshot Restore**.
    2. Select the backup snapshot you want to restore to.

    </div>
    <div label="Point-in-Time Restore">

    This feature lets you restore a cluster to a specific state from any time within the last 90 days.

    > **Note:**
    >
    > The **Point-in-Time Restore** feature is currently in beta.

>     To restore from any point in time within the backup retention, take the following steps:

Author: ```suggestion
    To restore from a specific point in time, take the following steps:
```
